lib/iris/coords.py

```diff
@@ -2704,7 +2704,7 @@ def _new_points_requirements(self, points):
             emsg = "The {!r} {} points array must be strictly monotonic."
             raise ValueError(emsg.format(self.name(), self.__class__.__name__))
 
-    @Coord._values.setter
+    @Coord._values.setter  # type: ignore[attr-defined]
```

Comment:
👍 I _think_ this is more sensible than giving `Coord._values` a setter specially?

@bjlittle @pp-mo any opinions?

Reply:
I suspect this is a bug in mypy (perhaps https://github.com/python/mypy/issues/1465) because `Coord._values` has a setter it inherits from `_DimensionalMetadata`:
https://github.com/bouweandela/iris/blob/7f84689237ee27a470d20265669857a043f23e68/lib/iris/coords.py#L222-L223

This code seems to pass the check though
```
    @property
    def _values(self):
        return super()._values

    @_values.setter
```
would you prefer that to ignoring the error?

Reply:
OK I looked into this more and I have a preferred suggestion:

```suggestion
    @property
    def _values(self):
        # Overridden just to allow .setter override.
        return super()._values

    @_values.setter
```

This is what we are already doing with the `Connectivity` class:

https://github.com/SciTools/iris/blob/1bea71c2bbc5aec1ee2be1a2b515989266f6a3d4/lib/iris/experimental/ugrid/mesh.py#L211-L217

Reply:
Hahah great minds think alike!

Reply:
Changed in ac4d44c and c3e5d8d (for the bounds)